benchopt/cli/main.py

```diff
@@ -200,6 +200,16 @@ def run(config_file=None, **kwargs):
     # Create the Benchmark object
     benchmark = Benchmark(benchmark)
 
+    if benchmark.min_version is not None:
+        from packaging.version import parse
+        from benchopt import __version__
+        if parse(__version__) < parse(benchmark.min_version):
+            raise RuntimeError(
+                "benchopt version is too old to run this benchmark. "
+                "Please update benchopt with `pip install -U benchopt` "
```

Comment:
This may not work if we require the dev version as in the deprecate `to_dict` issue?

Reply:
Yes, not sure what to do... We could say: `please update benchopt.` mais c'est peu actionable.